### PR TITLE
Add taurus symbol

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
Added taurus symbol

### CODE REVIEW

**Comment by DenverCoder1** [comment]
Please see the notes about formatting (https://github.com/DenverCoder1/LaTeX-Gboard-Dictionary/issues/5)

There must be tabs after the shortcut and after the symbol for it to work in Gboard

You can update it in the branch

Feel free to ask if you have any questions

**Comment by Pradumnasaraf** [comment]
@DenverCoder1 now you check, hope this time it's ok

**Comment by DenverCoder1** [comment]
Still not working, make sure there's a tab between (and no extra spaces) and a tab at the end of the line.

It's difficult to show in text, but if you copy this entire line exactly, it should work:

```
\taurus ♉ 
```

**Comment by Pradumnasaraf** [comment]
@DenverCoder1 plz check, still don't know if correct or even if io copied your code

**Comment by DenverCoder1** [comment]
Still not quite right, on GitHub, you can see the dots which represent spaces. It needs to be just a single tab for both like the examples above it.

image

File: dictionary.txt [hunk 1]
```diff
@@ -882,3 +882,4 @@ _rp	₎
 \ddagger	‡	
 \smalltilde	˜	
 -	−	
+\taurus ♉
```
Comment (DenverCoder1) [inline dictionary.txt:885]: Try "commit suggestion"

```suggestion
\taurus	♉	
```

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]

**Review by DenverCoder1 — APPROVED** [review]
Looks good! Thanks for contributing :tada:

**Pull request merged**